README.md

```diff
@@ -33,142 +33,184 @@ state of your service, interacting with the fundamental APIs:
BY1ui+cwoRD7yP+emPRC2SgzyvTWT4exxF2ci0J7qmkVGskSlaOYSzFFdL+aWbzqUv4KppZPNvrz7bUsKheXYqvnuwJVHERW2CHK8Pc4q3xEVJ2zHJvVym7qMyJLGqgstSrfa2isjgjlxZ90dAQ4LsTkBEhmSK0/PwlzBcc07XVeMLXbkzcB6HlqozuD+AhoIQtDQ298O4Ok081MwHv3lPXGrb0wCvP07ABHEa11XgG4tpPnqhaLW0Sz8DOXL84h7QSZlW9oEy7Mjzmdos3xOOc3owOcQnNE+QBzs1GezvX6BVROQClhUb6t9z0mhFLEeLhZ/X6PvwsmHT9+5iSTQrw1CQlTEckZPmfb6Xw1XLejF767UQ2Ou/l0izg76z0WSo11XsbKMBr+RK6ys1mZHL4hHVYn08Xg8+jU3jJo9fq7jjlx2JjmY3KzpbLzrKg2WpNhN+RxQTrlmaUscQdp4p+3ZT//aQcunViCgemKNNZ5GRx28MTNWus4HHQwf7AQ3mbM+VHFiKl/PuTT0bhnrhdS8GUg0ej5NhxDUG1f1N4NOqGPKJFt8axNVBNNyZlOst2M1j1IeMeUb29INQ1Oho+nsyWqyqlccXTmSd3PZS2PzpuMvbgHw0lVL4P0PEwXuME4M3mzFArrs68NpV+RaMabM3D8DBFidhJKDrlg11+eTuFAMT9NlbwG88IkSsVtqLbnIY875VJ+Eu0Dk0Kbmf9hjNCLPuePj/HpUmBEGF1OhAU3uSQMtXpGj+TdMRuswoeIlmxpk9mVfYaHhknKfo67lwcT5g2S3YXVSD5g2nKPhO24Pv3YPcm+FoEVTKVu2UmtkUnNEL8px+/IqTOuNJvBl7HSxHTlGw8TZuF0g1R3n6T+/eiLsFVp0YxSv63PwGKS6qsqzMRnXmhuNYGRYTlt0V+NoJQcItVQt57qWva3iHaVG1KbN4ClLaxrDqvNzjMGoMyNMPSQUvd0QpEordRmJ+xhXzWedQwuZxm6duRipmThmmtzIypuh8YwUa/7a+SOKUFMzcIqDmPHflWYGq0DQ5MBI5B4WNIiQ3V1JFvBlAtWs/tydaVjGxmLT7ihPUy2sUQSdupMlWKMu7xVU0JSeg2XuQwYwn/AovC1HHunyjXiKT6V92p350YlfqiqGlQ+vs6n0OrIc/D3XCAWaH80ZKVb2SrhlMuVeurSECm7+ixI3VZdDWupWLl1T5VFwky6gpmsWIhWhNJ+TN4lIu+wSekxzSizFebUrWWTLC7XAgsoRpqUl6vUMYj3HVYSjSNAeyVx5EWmcekobeSr+WP4YAEljierLlz+o6CzKFLmA172dAXAHdJye39CEc2P1B8sLaaqJMttjx4bzBRH5afbqYG1wXSUNkoMIkn+UYIvqpxFn7o/Y9UT23JTQjiw/8fBSTyrfRlvLbKQxC6aplLSVdK+pa8rLFp1a4KbLWUSgKvFLiAbWXVZx5eqciGkL20l2yb+QLqDz9xJjpIOOGLQcNMi0Fz+etsjSnW8RvWUC1Li3EQ/VeD14/jUp4PTteX2TBVXMkuCDZNM36buS/JVGlA+UHcAXNF1xcmpi5qjssvVDI+hlF76pWW2k6QRzFNJIvYGMimoUjgLJmerAb5NFwwMPbU2lu+cbxC3i9nNMR/DTdp4vsXza2ovYO1kEYROYpTsneOSTUeFFxlPgdGHZwuVOxIIL64UEDmaw8bk9alxqZgxWNtyJDA7AqQDtKL7Boz0R739j4nfsY8M7oaZ50agq5RJ4ChtPKb1HjQmc4/LAU3sYeIzvh8akqjWroZ4vvalH4YuMrZf6hhOQ9D/WoYe9RFD538+9bbfu6OpD2B15PhoZJ7Ul+S3XP02hFrEYyOq9NrH46xXCWxyL+q4thGnBR7x0FTDZEBqVpcHa0AqggxAEYkF7k3+i71uE7k/a6OqjK4YOv+i92/+Hp/Rv86tvf15MQLP0sb1MYr8LxV8WLp3mkDAnSKMaDrevR62q5t2ODe6z6PLqIAndt5wKvCjjm+UYqfeUEa4Pqr5iPu3gPlTjyVx+yVbjp4Thk4X1COkVvPD0SwzByhaL0/3nYB2xrmiuJRJ4GRJ3WPo9bnvURR21+b66lp7taeI4G0ifuGZwPFauvWgs6G6hQuAu7+rf4tCxw91TIf9VlMSetVabz0PzxrcJdaBeaJZOjzeg71yns67IIL/bNKOKhDi4phQxrnXWQ5Go8NqNO9OF+TEpMy7xPyBtjGJLZNSHwgGP5XmJ45gzYm6ZKjGICRVpnUPvUN70ZQmZRKAdZsbLzbcAirCIxzAQEV5UiZBF3RA7s0r9dknFtzjH7DmMRlduPKgCXGwP3ygzpFpjBJJBLmccFiilEkQQRKsk9svxxs1jwSxbNUNo6c8Q56t2zxCuzmeinqk+dQyKTQmzBTELmNjhFBirhfb7spbVuqSvgF8GRjYLalHjhe4Bk6EkMdof+Uqw1leBM2FjX8EwzDXG45CCwVrhdlTpEwCa7mxEDJ0NmQvCO9auk0Bx/SpBS1ty1F85PTDW45KeLjAbW0+1FSGspFK7RJ/OzmHeV+GxLdNrFqi98+TMgks0iZJ6s4QSGYkA1j6qdHnSZkEtr6+tDOrN9pq91kH0MWxUa8szUz+VCmTwCZtCMPU0dWiUHA2qSwvuWP/V8rmZobKk+scCr/XaBsg1fflSXDMUyoPz5cyCdzSXEoDzfkYZw44dBEvEfJDU09G8IJNxJn9kUpFuc/Kg5DK2XkyHBOyikPXE/u5+GVpQ7Uv2jwV4PdfSgHO+eJvIdR+kUIM6rxaOnqOOSEF8NtSJoHhVaDygdweOioRo9+RNr+oy6QAfZehEDJr+p+JWjnv29sJfI2USdDTdMorGa+SMgla2yefxaxCreejPrfY5IWBuWM/4qegN+X0GecxbmuwyJUZWz5ahy31UbnwKQnnfwh9FwSVstH6RXv3jTfeeOONN95444033iiK/wDpWyMpdIpCEgAAAABJRU5ErkJggg==
```

Comment:
Can we keep it shorter in Readme?

Reply:
Ok. I will find a smaller icon to replace it.